src/implementations/deploy/document-store/document-store.ts

```diff
@@ -13,9 +13,10 @@ export const deployDocumentStore = async ({
   network,
   gasPriceScale,
   dryRun,
+  passedOnWallet,
```

Comment:
afterwards could you put a note there to indicate that `wallet` variable being passed on is only used in create implementation?
same for deploy token reg

Reply:
ok